feedingwebapp/src/Pages/Settings/Settings.jsx

```diff
@@ -29,7 +46,9 @@ const Settings = (props) => {
         return (
           <CustomizeConfiguration
             startingMealState={MEAL_STATE.R_MovingAbovePlate}
-            paramNames={[ABOVE_PLATE_PARAM_JOINTS]}
+            paramNames={abovePlateParamNames}
+            getEndEffectorPose={false}
+            getParamValues={[getJointPositionsFromRobotStateResponse]}
```

Comment:
Consider wrapping these in `useMemo` as well to avoid re-making them every re-render

Reply:
It only did one set parameter call without this, so I don't think it results in re-calling useEffects.